cluster/src/main/java/com/hartwig/pipeline/calling/structural/gridss/stage/Driver.java

```diff
@@ -21,21 +23,20 @@
     private final String referenceBamPath;
     private final String tumorBamPath;
 
-    public Driver(final String assemblyBamPath, final String referenceGenomePath, final String blacklistBedPath,
-            final String gridssConfigPath, final String repeatMaskerDbBed, final String referenceBamPath, final String tumorBamPath) {
+    public Driver(final ResourceFiles resourceFiles, final String assemblyBamPath, final String referenceBamPath, final String tumorBamPath) {
         super("gridss.driver", OutputFile.GZIPPED_VCF);
         this.assemblyBamPath = assemblyBamPath;
-        this.referenceGenomePath = referenceGenomePath;
-        this.blacklistBedPath = blacklistBedPath;
-        this.gridssConfigPath = gridssConfigPath;
-        this.repeatMaskerDbBed = repeatMaskerDbBed;
+        this.referenceGenomePath =  resourceFiles.refGenomeFile();
+        this.blacklistBedPath = resourceFiles.gridssBlacklistBed();
+        this.gridssConfigPath = resourceFiles.gridssPropertiesFile();
+        this.repeatMaskerDbBed = resourceFiles.gridssRepeatMaskerDbBed();
```

Comment:
I'd be inclined to just store `resourceFiles` as an instance variable, the method names on it are descriptive enough.